### DISCUSSION TITLE
Limiting Service Event editing to just Return to Service QA

### DISCUSSION THREAD

**Discussion by nsmela** [qatrackplus/qatrackplus#598] [Q&A] [answered]
Hi there,

We're giving our Physicists Assistants access to service events to view, add, change or remove Return To Service QA. I'm struggling to figure out the permissions set to allow them to do this without giving them access to change details for the servicing (work done, service areas, etc) although we're debating on having physics include QA as part of the "Lost Time" entry.

Is it possible to limit access to service event fields while having access to the Return To Service QA fields?

**Comment by randlet** [comment, marked as answer]
Hi @nsmela sorry for the slow response. Unfortunately the permissions are not fine grained enough to make this possible.

**Reply by nsmela** [reply]
No worries regarding the response time.  

Would it be simple process to add this type of permission as a new feature?

**Reply by randlet** [reply]
Now that I think about it a bit more, I think actually that maybe the permissions are available, they're just not put into practice.  The permissions for adding /deleting return to service qa are there and accounted for, but everything assumes the user also has permission to create/update service events as well.  So the service event html template files, and some of the form processing views would likely need to be adapted to account for the possibility of having the relevant RTS permissions, but not the Service Event permissions.
